gdk/commands/component/BuildCommand.py

```diff
@@ -14,13 +11,15 @@
 from gdk.build_system.BuildSystem import BuildSystem
 from gdk.build_system.Zip import Zip
 from gdk.commands.Command import Command
+from gdk.commands.component.transformer.BuildRecipeTransformer import BuildRecipeTransformer
 
 
 class BuildCommand(Command):
     def __init__(self, command_args) -> None:
         super().__init__(command_args, "build")
         self.project_config = project_utils.get_project_config_values()
         self.supported_build_sytems = project_utils.get_supported_component_builds()
+        self.build_recipe_transformer = BuildRecipeTransformer(self.project_config)
```

Comment:
I understand passing the entire project config makes it easy but it opens the door for internally coupling to the self.project_config structure. Could we just pass it what it needs?

Reply:
BuildReciperTransformer needs 9 properties of `self.project_config`. So, I think we can pass the full object for now

Reply:
We should be able to pass less as we split this in the future. It is ok for now